screen-orientation.opam

```
@@ -15,4 +15,11 @@ depends: [
   "gen_js_api"
   "js_of_ocaml" {>= "3.8.0"}
   "js_of_ocaml-ppx" {>= "3.8.0"}
-  ]
+  "lwt_ppx" {>= "2.0.1"}
+  "promise_jsoo"
+  "promise_jsoo-ppx"
+  "promise_jsoo_lwt"
+]
+pin-depends: [
+  [ "promise_jsoo_lwt" "git+https://github.com/mnxn/promise_jsoo"]
```

It seems that we need to explicitly specify a version:

```suggestion
  [ "promise_jsoo_lwt.dev" "git+https://github.com/mnxn/promise_jsoo"]
```